electron/src/updater/updateServer.ts

```diff
@@ -0,0 +1,82 @@
+"use strict";
```

Comment:
Yes, for the Mac updater. Electron.autoUpdater expects a JSON response containing a url to the downloadable asset, but our backend api downloads the asset directly. This server simply immediately returns the necessary json response:
```
{
    url: "url to downloadable update form pxt backend"
}
```

If we remove this file, we need a new route in our backend to return the above response. I think an extra file in the app is much better than another query to the backend (which can fail, takes some time, etc)